docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+web:
+    build: .
```

Comment:
:+1: for move the `Dockerfile` in a subdirectory (`docker/php/Dockerfile` for example). And keep the `docker-compose.yml` here.